src/Tizen.NUI.Components/Controls/RecyclerView/CollectionView.cs

```diff
@@ -673,7 +729,7 @@ public override void OnRelayout(Vector2 size, RelayoutContainer container)
             base.OnRelayout(size, container);
 
             wasRelayouted = true;
-            if (needInitalizeLayouter) Init();
+            if (needInitalizeLayouter) ReinitializeLayout();
```

Comment:
do you want to change this as follows?
```
if (needInitalizeLayouter)
{
    ReinitializeLayout();
}
```

Reply:
oh I missed it. thank you for noticing. will fix right now.